### DISCUSSION TITLE
Always Complete, Never Finished

### DISCUSSION THREAD

**Discussion by baileythegreen** [widdowquinn/pyani#308] [General]
A while back I mentioned a talk I saw on documentation writing philosophy. I think it contains a lot of good ideas, and the video is available on [youtube](https://www.youtube.com/watch?v=Wc7n7uIg4AM&list=PLZAeFn6dfHpkCk5Fw5yLFGTqd1OJoaXhR&index=3).

The main idea is to always try to improve the documentation for what the code is at that time, through incremental fixes/additions. I think it's similar to the idea of atomic commits—the documentation should always be in a usable state. ('Usable' here has a slightly different definition.)

**Comment by widdowquinn** [comment]
Thanks - I'll take a look at the video.
